### ISSUE TITLE
How to Create a Blank Layer (Blank Provider)?

### ISSUE THREAD

**Issue by 88madri** [pauldendulk/Mapsui#65]
Hi I want to create a blank map but with a valid return of coordinates 
with WorldToScreen or ScreenToWorld 

I want this extents and ESRI Code

```
    private const double DEFAULT_X_MIN = -180;
    private const double DEFAULT_X_MAX =  180;
    private const double DEFAULT_Y_MIN = -90;
    private const double DEFAULT_Y_MAX =  90;
    private const int WGSG84_ESRI_CODE = 4326;
```

This could be possible??

**Comment by pauldendulk** [comment]
This was not possible at the time the question was asked. Some improvements were made related to this. I'll if is possible now.

**Comment by pauldendulk** [comment]
Fixed with commit a4dde4c1069cd0d212c0672c8dfb6a24e5b554a2
Look at EmptyMapSample
